larpandora/LArPandoraInterface/LArPandoraHitCollectionTool.h

```diff
@@ -0,0 +1,24 @@
+/**
+ *  @file  larpandora/LArPandoraInterface/Tools/LArPandoraHitCollectionTool.h
+ * 
+ *  @brief Define class for hit collection tools
+ * 
+ */
+#ifndef LAR_PANDORA_HIT_COLLECTION_TOOL_H
+#define LAR_PANDORA_HIT_COLLECTION_TOOL_H
+
+#include "larpandora/LArPandoraInterface/LArPandoraHelper.h"
+
+#include "art/Framework/Principal/Event.h"
+
+namespace HitCollectionTools {
+
+  class HitCollectionTool
```

Comment:
Probably want to call this IHitCollectionTool in keeping with other pure interfaces in larpandora.